ci-operator/config/netobserv/flowlogs-pipeline/netobserv-flowlogs-pipeline-main.yaml

```diff
@@ -18,14 +34,32 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.18"
 resources:
   '*':
     limits:
       memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: qe-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      PATCH_FLOWLOGS_IMAGE: pipeline:flowlogs-pipeline
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_FILTERS: NonPreRelease;Serial;Disruptive;~59746&
+      TEST_SCENARIOS: Network_Observability
+      TEST_TIMEOUT: "35"
+    test:
+    - ref: netobserv-install-operator
+    - ref: openshift-extended-test
+    - ref: openshift-e2e-test-qe-report
+    workflow: cucushift-installer-rehearse-aws-ipi
+  timeout: 8h0m0s
```

Comment:
Is there a reason to use such a huge timeout here ?

I feel it would be more reasonnable to use 2 or 3 times the current time needed to run the tests at max and increase these in the future if we start getting timeouts. WDYT ?